test/model/toRestHandlers.test.ts

```diff
@@ -558,3 +562,170 @@ describe('DELETE /users/:id', () => {
     })
   })
 })
+
+describe('GET /todos/:id', () => {
+  it('handles a GET request to get a single entity', async () => {
```

Comment:
We should put these tests into respective `describe` blocks that already exist. We may have to rename the model name to `user` to be consistent. What's important is to name the test case properly, so it's clear what use case we're asserting.

```js
it('handles a GET request to get an entity by a numeric primary key', ...)
```

Reply:
The `user` model name already exists. I added the `todos` model with a numeric primary key to add new assertions. That's the reason for this new `describe` block.

Reply:
Got it. I've moved this entire scenario into its own test suite so it's easier to read it.